project4/styles.css

```diff
@@ -0,0 +1,300 @@
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--color-black);
+    color: whitesmoke;
+    font-family: montserrat-alternates, sans-serif;
+    margin: 0;
+}
+
+header img,
+header div {
+    display: inline-block;
+}
+
+header div {
+    text-align: center;
+}
+
+header h3 {
+    margin: 0;
+}
+
+h1,
+h2,
+h3 {
+    color: var(--color-pink);
+}
+
+h1,
+h2 {
+    font-family: bigsmalls-bold, sans-serif;
+}
+
+h3,
+h4 {
+    font-family: bigsmalls-regular, sans-serif;
+
+}
+
+h1 {
+    font-size: 15px;
+    line-height: 1.5;
+    margin: 0;
+}
+
+h2 {
+    font-size: 50px;
+}
+
+p {
+    font-family: montserrat-alternates, sans-serif;
+}
+
+nav ul {
+    font-family: bigsmalls-regular, sans-serif;
+    list-style: none;
+    padding-left: 0;
+    text-align: center;
+}
+
+nav li {
+    display: inline-block;
+    line-height: .8;
+    max-width: 75px;
+
+}
+
+nav li a {
+    color: var(--color-white);
+    font-size: 10px;
+    line-height: 0;
+    /* margin-right: 2px; */
+    margin-left: 6px;
+    padding: 5px;
+    text-decoration: none;
+}
+
+nav li a:hover,
+nav li a:focus {
+    background-color: var(--color-pink);
+    border-radius: 3px;
+}
+
+img {
+    max-width: 100%;
+    overflow-x: hidden;
+}
+
+
+/* nav li+li {
+    border-left: 1px solid var(--color-white);
+} */
+
+hr {
+    display: block;
+    border-radius: 10px;
+    border-style: none;
+    border: 1.5px solid var(--color-white);
+    margin: 0;
+    margin-left: 30px;
+    text-shadow: none;
+    width: 500px;
+}
+
+section {
+    padding: 100px 30px;
+    /* just for demo purposes */
+}
+
+
+svg path {
+    fill: var(--color-pink);
+}
+
+
+
+#speakers h3 {
+    font-size: 30px;
+}
+
+#exhibitions-learning p {
+    width: 75%;
+}
+
+
+input[type="email"]:focus {
+    border: 3px solid var(--color-pink);
+}
+
+input[type="email"]:required:valid {
+    background-color: var(--color-pink);
+}
+
+.cta-tickets {
+    background-color: var(--color-white);
+    border-radius: 50%;
+    color: var(--color-black);
+    display: block;
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 11px;
+    line-height: 15px;
+    padding-bottom: 18px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-top: 18px;
+    position: absolute;
+    right: 20px;
+    text-align: center;
+    text-decoration: none;
+    top: 30px;
+}
+
+.logo {
+    left: 0;
+    top: 30px;
+    width: 35%;
+}
+
+.theme {
+    color: var(--color-white);
+    font-size: 8px;
+    letter-spacing: 3px;
+    line-height: 1;
+}
+
+
+
+
+.main-title {
+    padding-bottom: 80px;
+    padding-left: 30px;
+    padding-top: 80px;
+}
+
+.main-title p {
+    color: var(--color-pink);
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 60px;
+    line-height: 25px;
+}
+
+.first {
+    border: 1.5px solid var(--color-pink);
+}
+
+.speaker {
+    border-top-left-radius: 30%;
+    border-bottom-right-radius: 30%;
+    width: 100%;
+
+}
+
+.andrea,
+.sj,
+.arielle {
```

Comment:
Instead of giving them unique names to the speakers, consider treating these as shared components (so, all having the _shared_ class of `speaker-container` or something). (See additional note about this under media queries.)